src/include/daos/common.h

```diff
@@ -119,8 +119,12 @@ char *daos_key2str(daos_key_t *key);
 				daos_key2str(key)
 #endif
 
-#define DF_RECX			"["DF_U64"-"DF_U64"]"
-#define DP_RECX(r)		(r).rx_idx, ((r).rx_idx + (r).rx_nr - 1)
+#define DF_RECX			"["DF_U64"-"DF_U64", "DF_U64"]"
+#define DP_RECX(r)		(r).rx_idx, ((r).rx_idx + (r).rx_nr - 1), ((r).rx_nr)
```

Comment:
(style)  line over 80 characters
(style)  Macros with complex values should be enclosed in parentheses